packages/pilot/src/containers/AddRecipient/ConfigurationStep/Anticipation/index.js

```diff
@@ -39,17 +40,26 @@ const anticipationModelOptions = ({
     value: 'automatic_dx',
   }
 
+  const custom = {
+    name: 'Customizado',
+    value: 'custom',
+  }
+
   const availableOptions = [
     manualByVolume,
     automaticByVolume,
   ]
 
-  if (canConfigureAnticipation) {
+  if (!canConfigureAnticipation) {
     availableOptions.push(automaticAtDays10And25)
+    availableOptions.push(automaticDX)
+    availableOptions.push(custom)
   }
 
   if (canConfigureAnticipation && maximumAnticipationDays >= 31) {
+    availableOptions.push(automaticAtDays10And25)
     availableOptions.push(automaticDX)
+    availableOptions.push(custom)
   }
```

Comment:
o conteudo interno desses 2 `ifs` é igual, poderia ser feita uma validação que utilize dos dois
```suggestion
if (
!canConfigureAnticipation
|| (canConfigureAnticipation && maximumAnticipationDays >= 31)
) {
```

Reply:
Verdade!